example.json

```diff
@@ -1,5 +1,6 @@
   // apply to course.json for menu header
   // apply to contentObject.json for page header
+  // apply to components.json for component
```

Comment:
In all instances, the button is appended to the header.
e.g `// apply to components.json for component header`

Reply:
Done in [775b1b5](https://github.com/cgkineo/adapt-scrollPrompt/pull/4/commits/775b1b578626b42bcb85520f605a1a84f019cec5)